src/graphql/admin/mongoose_graphql_account_admin_query.erl

```diff
@@ -0,0 +1,74 @@
+-module(mongoose_graphql_account_admin_query).
+
+-export([execute/4]).
+
+-ignore_xref([execute/4]).
+
+-include("../mongoose_graphql_types.hrl").
+
+-import(mongoose_graphql_account_helper, [format_result/2, make_error/3]).
+
+execute(_Ctx, _Obj, <<"listUsers">>, Args) ->
+    list_users(Args);
+execute(_Ctx, _Obj, <<"countUsers">>, Args) ->
+    count_users(Args);
+execute(_Ctx, _Obj, <<"countActiveUsers">>, Args) ->
+    get_active_users_number(Args);
+execute(_Ctx, _Obj, <<"checkPassword">>, Args) ->
+    check_password(Args);
+execute(_Ctx, _Obj, <<"checkPasswordHash">>, Args) ->
+    check_password_hash(Args);
+execute(_Ctx, _Obj, <<"checkUser">>, Args) ->
+    check_user(Args).
+
+% Internal
+
+-spec list_users(map()) -> {ok, [{ok, binary()}]}.
+list_users(#{<<"domain">> := Domain}) ->
+    Users = mongoose_account_api:list_users(Domain),
+    Users2 = lists:map(fun(U) -> {ok, U} end, Users),
+    {ok, Users2}.
+
+-spec count_users(map()) -> {ok, non_neg_integer()}.
+count_users(#{<<"domain">> := Domain}) ->
+    Users = mongoose_account_api:list_users(Domain),
```

Comment:
I think we need a new API function here that will call `get_vh_registered_users_number`, otherwise this might be surprisingly heavy for the server, even if the user turned on the users number estimate.